### PR TITLE
Use external GitConnector in case there's no one in tracopt

### PR DESCRIPTION
**Pull request by ghost**
Import GitConnector from `tracopt` _or_ `tracext`, making it work with Trac 0.12

### CODE REVIEW

**Comment by csnover** [comment]
To confirm, this is for backwards support to 0.12?

File: github/github.py [hunk 1]
```diff
@@ -6,7 +6,10 @@
 from trac.versioncontrol.web_ui.changeset import ChangesetModule
 from trac.web.api import IRequestHandler, RequestDone
 from trac.wiki import IWikiSyntaxProvider
-from tracopt.versioncontrol.git.git_fs import GitConnector
+try:
+    from tracopt.git.git_fs import GitConnector
```
Comment (csnover) [inline github/github.py]: this isn’t the right import

**Comment by ghost** [comment]
> this isn’t the right import

Whoops, sorry.

> To confirm, this is for backwards support to 0.12?

Yep. I've pushed amending commit with fixed import and more detailed commit message.

**Comment by ghost** [comment]
ping? I've got another fix in queue, waiting for this one to be accepted :)

**Comment by ghost** [comment]
OK, this obviously isn't needed.

**Pull request closed without merging**